media-ranker/app/views/albums/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="col-lg-12">
+  <h2>Edit Album</h2>
```

Comment:
You might be able to create a shared form for editing any media type, and likewise for creating any media type.